src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -18,9 +18,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
   <PropertyGroup>
     <!-- Override the TargetFrameworkVersion specified in build.props. This project must target
-         .NET 4.6 because the Sarif.Validator project, upon which this project depends, targets
+         .NET 4.5.2 because the Microsoft.Json.Schema package, upon which this project depends, targets
```

Comment:
JSchema can almost certainly build against 4.5.1; I'll fix it.

I don't understand your second sentence. It seems to recapitulate the comment ("Override the TargetFrameworkVersion...").